system-int-test/test/cmr/system_int_test/ingest/collection/collection_translation_test.clj

```diff
@@ -121,7 +121,8 @@
       (is (= 200 status))))
 
   (testing (format "Translating umm-json to iso19115 produces the right codelist")
-    (let [input-format :umm-json
+    (let [input-format {:format :umm-json
+                        :version "1.18.2"}
```

Comment:
why 1.18.2 instead of 1.18.3?

Reply:
same reason as above.  I didn't add these test files and if we don't add the version, every time someone creates a new UMM-C version we would have to update test files.